python/shipDigiReco.py

```
@@ -146,6 +186,340 @@ def reconstruct(self) -> None:
+        self.fACTSArray.clear()
+        self.fACTSVertexArray.clear()
+        self.fitTrack2MC.clear()
+        self.fPartArray.clear()
+        self.fTrackletsArray.clear()
+
+        output_tracks, vertices, track_hit_indices = self.actsTracks()
+
+        vector_ptr = ROOT.addressof(self.fACTSArray)
+        vertex_vector_ptr = ROOT.addressof(self.fACTSVertexArray)
+
+        for i, (_, hit_indices) in enumerate(zip(output_tracks, track_hit_indices)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)